server/controllers/SolutionController.js

```diff
@@ -59,9 +63,42 @@ exports.create = async (req, res) => {
 
 exports.edit = async (req, res) => {
   try {
-    const { id } = req.params;
-    const { challengeId, userId } = req.body;
-    await Solution.update({ challengeId, userId }, { where: { id } });
+    const { file } = req;
+    const { challengeId, userId, description, title, id } = req.body;
```

Comment:
Are we ensuring that only the owner of the solution can make edits?

Reply:
Not yet, I'll add a todo for when the auth is properly working. We also need to consider the admin's ability to edit.